lib/rspec/core/fail_fast_handler.rb

```diff
@@ -0,0 +1,17 @@
+module RSpec
+  module Core
+    module FailFastHandler
```

Comment:
There's no docs for this module which is what is putting off the coverage, mark them with `# @api private` if they're not intended for public usage.